playbooks/common/openshift-node/config.yml

```diff
@@ -139,6 +139,8 @@
 - name: Set scheduleability
   hosts: oo_first_master
   vars:
+    openshift_nodes: "{{ hostvars
+                         | oo_select_keys(groups['oo_nodes_to_config']) }}"
```

Comment:
Thanks! QE noticed that too.. https://bugzilla.redhat.com/show_bug.cgi?id=1241342.  I'll update it and submit it for testing.